astropy/io/ascii/CDSColumn.py

```diff
@@ -0,0 +1,529 @@
+"""
+   G.Landais (CDS) 28 nov 2015
+   CDSColumn, used to Generate ReadMe and CDS standardized tables (in ASCII aligned columns)
```

Comment:
Based on all the helpful comments on this PR, you could entirely remove this file or (even better) squash the first few commits when you added this file (and some others). Refer to [How to Squash](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#how-to-squash) if you haven't used `squash` before. @Suyog7130, you already mentioned that you will be removing any changes you directly included from `cdspyreadme` -- I just wanted to provide information regarding how to squash so that you are on track!